README.md

```diff
@@ -0,0 +1,106 @@
+![linkerd2][logo]
+
+[![Build Status][ci-badge]][ci]
+[![GitHub license][license-badge]](LICENSE)
+[![Slack Status][slack-badge]][slack]
+
+This repo contains the transparent proxy component of [Linkerd2][linkerd2].
+While the Linkerd2 proxy is heavily influenced by [the Linkerd 1.X
+proxy][linkerd1], it comprises an entirely new codebase implemented in the
+[Rust programming language][rust].
+
+This proxy's features include:
+
+* Transparent, zero-config proxying for HTTP, HTTP/2, and arbitrary TCP protocols.
+* Automatic [Prometheus][prom] metrics export for HTTP and TCP traffic;
+* Transparent, zero-config WebSocket proxying;
+* Opportunistic TLS;
```

Comment:
I would phrase this as "Experimental automatic TLS support (temporarily opportunistic)"